src/api/team-typedefs.ts

```diff
@@ -0,0 +1,23 @@
+export const teamTypedefs = `
```

Comment:
Labels instead of straight strings for a potential future where we need translated affiliation or team names.